tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -54,7 +54,7 @@ DomainCreator:
         ExciseWithBoundaryCondition: Worldtube
       UseLogarithmicMap: false
     Envelope:
-      UseProjectiveMap: false
+      RadialDistribution: Logarithmic
```

Comment:
Keep the old value of "Linear" or "Projective" I think (same in other input files)